app/frontend/wallet/shelley/shelley-ledger-crypto-provider.ts

```diff
@@ -114,7 +106,7 @@ const ShelleyLedgerCryptoProvider = async ({
   const ledger = new Ledger(transport)
   const derivationScheme = derivationSchemes.v2
 
-  const version = await ledger.getVersion()
+  const {version} = await ledger.getVersion()
 
   const getVersion = (): string => `${version.major}.${version.minor}.${version.patch}`
```

Comment:
Not related to this PR, but why is this a function and not just called `versionString` without being a function?

Reply:
it's an outward facing function, part of the general crypto provider interface, and other crypto providers also have a getter for the version - trezor/mnemonic crypto provider should be able to migrate to the version as a property as well, but probably not worth it